src/App.jsx

```diff
@@ -1,14 +1,30 @@
 import * as React from 'react'
+import Card from './Card'
 import Header from './Header'
 import Pagination from './Pagination'
+import trivia from './api.php.json'
+import './App.css'
+import Navigation from './Navigation'
+
+const navText = ['Home', 'About', 'Settings']
 
 export default () => {
   return (
-    <div>
-      <Header isBackButtonVisible>Header with button</Header>
-      <Pagination isLeftDisabled text="1/34" />
-      <Pagination text="4/34" />
-      <Pagination isRightDisabled text="34/34" />
+    <div className="App">
+      <Navigation navButtons={navText} clicked="About" />
+
+      {trivia.map(el => {
+        const { category, question, correct_answer, incorrect_answers } = el
+        return (
+          <Card
+            key={correct_answer}
+            category={category}
+            question={question}
+            correctAnswer={correct_answer}
+            tagList={incorrect_answers}
+          />
+        )
+      })}
```

Comment:
You can do this even shorter:
```suggestion
      {trivia.map(({ category, question, correct_answer, incorrect_answers }) => (
          <Card
            key={correct_answer}
            category={category}
            question={question}
            correctAnswer={correct_answer}
            tagList={incorrect_answers}
          />
        )
      )}
```